pkg/restapi/verifier/operation/operations.go

```diff
@@ -182,6 +186,35 @@ func (o *Operation) getProfileHandler(rw http.ResponseWriter, req *http.Request)
 	commhttp.WriteResponse(rw, profile)
 }
 
+// DeleteVerifierProfile swagger:route DELETE /verifier/profile/{id} verifier deleteProfileReq
+//
+// Deletes verifier profile.
+//
+// Responses:
+// 		default: genericError
+//			200: emptyRes
+func (o *Operation) deleteProfileHandler(rw http.ResponseWriter, req *http.Request) {
+	profileID := mux.Vars(req)["id"]
+
+	err := o.profileStore.DeleteProfile(profileID)
+
+	if err != nil {
+		if strings.Contains(err.Error(), storage.ErrValueNotFound.Error()) ||
+			strings.Contains(err.Error(), "Not Found: deleted") {
+			commhttp.WriteErrorResponse(rw, http.StatusNotFound,
+				fmt.Sprintf(verifierProfileNotFoundErrMsg, profileID, err.Error()))
+
+			return
+		}
+
+		commhttp.WriteErrorResponse(rw, http.StatusBadRequest, err.Error())
+
+		return
+	}
+
+	rw.WriteHeader(http.StatusOK)
```

Comment:
@grace0925 Note that http.StatusOK is automatically written if you don't write anything else, so you don't have to add it normally